### PR TITLE
28 change seek help page + styling

### PR DESCRIPTION
**Pull request by arfamomin**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Updated styling to match Kevin's designs 
- Removed nav for California/National and made into buttons on main Seek Help page  
- Added logic for filtered by level 
- Updating _some_ styling on right panel resource cards

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

Screenshot 2024-12-05 at 12 16 45 AM
Screenshot 2024-12-05 at 12 28 40 AM

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- SeekHelp folder for logic and styling updates 
- Resource component for styling updates on cards

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- I didn't fully do what is on the figma for the fields (i.e. address, phone number, etc..) because it seems like different resources have different fields and that might affect how things are organized. Think we could talk about what this should look like a little more (so rn I just stacked the fields); Also I think Kevin is changing the design for the fields based on the comments in the Figma?
- the collapsed summary text cutoff/splice is being hardcoded (so it looks different on different screen sizes); I didn't know if it's worth it to dynamically calculate the line width for the text since all the prison tablets are the same size? so I guess we could adjust that once we test on proper dimensions
- didn't change the carrot thingy because I think we said that it was good?

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @philipye314

### CODE REVIEW

File: src/screens/SeekHelp/index.tsx [hunk 1]
```diff
@@ -1,25 +1,179 @@
...
+                    styles.checkButton,
+                    selectedJurisdiction.includes(level)
+                      ? styles.selectedCheckButton
+                      : null,
+                  ]}
+                >
+                  {selectedJurisdiction.includes(level) && <Check />}
+                </View>
+                <Text style={styles.buttonText}>{level}</Text>
+              </TouchableOpacity>
```
Comment (philipye314) [inline src/screens/SeekHelp/index.tsx:121]: I changed it for you - but I basically swapped the inner view and touchable opacity. generally we want buttons to take up as much space as possible in the container such that things are more clickable (and thereby accessible) - this is why I try and refrain from using margins and normally use padding! You can see that adding padding or simply swapping react components around can make the clickable area a lot bigger and more intuitive to use.

**Review by philipye314 — APPROVED** [review]
looks pretty good overall! Just made one small comment. great work on the styling Arfa!
